### DISCUSSION TITLE
does it make sense to add a test project that mirrors the playground functionality?

### DISCUSSION THREAD

**Discussion by gotmike** [betalgo/openai#93] [Ideas]
when using the playground it is sometimes tedious to comment/uncomment code. seems like maybe a test project would be easier?

alternately, of course, could build some sort of selection logic into the playground.

**Comment by kayhantolga** [comment]
Good one! I agree. We need a test project different than Playground. I will create a branch soon so we can add tests as much as we want.

**Comment by kayhantolga** [comment]
https://github.com/betalgo/openai/tree/feature/Tests 

I have created a branch with 2 new empty test projects. 
I think Integration tests should be like playground tests. They should make a request to openAI services. I have never done integration tests this way but we can give it a try and see if they will become something useful.
Feel free to create tests and PR if you want. I will be busy this week but I may join you from time to time.
